lib/rules/jsx-wrap-multilines.js

```diff
@@ -98,9 +98,19 @@ module.exports = {
       const previousToken = sourceCode.getTokenBefore(node);
       const nextToken = sourceCode.getTokenAfter(node);
 
-      return isParenthesised(node) &&
-        previousToken.loc.end.line === node.loc.start.line &&
```

Comment:
it seems like the effective change here is changing `&&` to `||`?

Reply:
Yes. I find mixing `&&` and `||` harder to read but if you prefer it that way I can switch it back.

Reply:
Nah this is good, just making sure i understand.

Reply:
Apparently I didn't quite understand how the nodes worked with whitespace. I had to break this up into separate functions since the newlines were getting attached to the previous token.